src/preamble.js

```diff
@@ -295,9 +295,17 @@ function updateGlobalBufferAndViews(buf) {
   Module['HEAPF32'] = HEAPF32 = new Float32Array(buf);
   Module['HEAPF64'] = HEAPF64 = new Float64Array(buf);
 #if WASM_BIGINT
+#if MIN_SAFARI_VERSION < 150000
+#if !POLYFILL
+    assert(false, "Cannot support Safari 14 with WASM_BIGINT unless POLYFILL is also set");
+#else // POLYFILL
+    #include "polyfill/bigint64array.js"
+#endif // POLYFILL
```

Comment:
Can you move this pollyfill into `src/shell.js` along with all other ones?


The idea is that the locations that use the polyfills items don't need to know about the polyfills.. they can just assume its existence.

Reply:
(then you don't need the indented include, or the patch to parseTools.py)

Reply:
I will split off the patch to `parseTools.js` anyways because I think it's a correct patch.

Reply:
Sounds good!